corehq/apps/consumption/models.py

```diff
@@ -1,14 +1,44 @@
 from dimagi.ext.couchdbkit import DecimalProperty, Document, StringProperty
+from django.db import models
 
 from corehq.apps.cachehq.mixins import CachedCouchDocumentMixin
+from dimagi.utils.couch.migration import SyncCouchToSQLMixin, SyncSQLToCouchMixin
 
 TYPE_DOMAIN = 'domain'
 TYPE_PRODUCT = 'product'
 TYPE_SUPPLY_POINT_TYPE = 'supply-point-type'
 TYPE_SUPPLY_POINT = 'supply-point'
 
 
-class DefaultConsumption(CachedCouchDocumentMixin, Document):
+class SQLDefaultConsumption(SyncSQLToCouchMixin, models.Model):
+    type = models.CharField(max_length=32, null=True)  # 'domain', 'product', 'supply-point-type', 'supply-point'
```

Comment:
is this a fixed list of values? Could be good to add them as `choices`

Reply:
0589b08f8e3